Products/ZenHub/PBDaemon.py

```diff
@@ -1100,6 +1100,12 @@ def buildOptions(self):
                                type='int',
                                help='How often to ping zenhub')
 
+        self.parser.add_option('--ping-perspective',
+                               dest='pingPerspective',
+                               help="Enable or disable ping perspective",
+                               default=True,
+                               action='store_true')
```

Comment:
This option configuration fails with an error if I try to specify `--ping-perspective=False` on the command line.